httpbakery/client_test.go

```diff
@@ -34,6 +33,8 @@ type ClientSuite struct {
 
 var _ = gc.Suite(&ClientSuite{})
 
+var macaroonCurrentVersion = bakery.MacaroonVersion(bakery.LatestVersion)
```

Comment:
The MacaroonVersion function derives the correct macaroon version given a the bakery version. They're not in lockstep, which is why we call the function to do this (and why macaroonCurrentVersion is a variable not a constant). Were you thinking this was just a type conversion, perhaps?